cli/cli_kafka_runner_test.go

```diff
@@ -0,0 +1,343 @@
+package cli
+
+import (
+	"bufio"
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+	"os"
+	"strings"
+	"testing"
+
+	"github.com/featurebasedb/featurebase/v3/cli/internal"
+	"github.com/featurebasedb/featurebase/v3/cli/kafka"
+	"github.com/featurebasedb/featurebase/v3/errors"
+	"github.com/featurebasedb/featurebase/v3/logger"
+	"github.com/stretchr/testify/require"
+)
+
+// These variables are used throught tests to refer to resouces needed for the
+// tests (i.e. where to find the kafka service)
+var (
+	pilosaHost     string
+	pilosaTLSHost  string
+	pilosaGrpcHost string
+	kafkaHost      string
+	registryHost   string
+	certPath       string
+)
+
+// The variables above are set here. If local is set to true, the tests assume
+// all services are running locally. Otherwise, we use the name of the
+// containers that these tests will have access to in CI.
+func init() {
+	local := true
```

Comment:
I think if we can avoid using `init()` functions that's best. Even if it's just a `setup()` function which is then called in the test run.

The hard-coded `local` here obviously won't work, but I'm sure you know that. Just a reminder that test should be configured outside of the code.

Maybe look at the way config is setup here: https://github.com/FeatureBaseDB/featurebase/blob/master/dax/controller/sqldb/test.go